tmt/steps/provision/__init__.py

```diff
@@ -222,14 +222,17 @@ def _ssh_options(self, join=False):
         if self.port:
             options.extend(['-p', str(self.port)])
         if self.key:
-            options.extend(['-i', self.key])
+            options.extend(['-i', f'"{self.key}"' if join else self.key])
         return ' '.join(options) if join else options
```

Comment:
IMHO it is safer to run shlex.quote on whole options. self.key can contain double quote

Reply:
Good point, substituted with `shlex.quote`.